frontend/src/renderer/components/Charts/LineChart.svelte

```diff
@@ -65,18 +65,14 @@
               curve: 'smooth',
           },
           toolbar: {
-              // Hamburger menu which has exports such as CSV etc.
+              // Hamburger menu which has exports such as PNG and SVG, CSV is explicitly hidden via css
               show: true,
               tools: {
                   download: true,
               },
               export: {
-                  csv: {
-                      filename: 'SpectralLineChart.csv',
-                      columnDelimiter: ',',
-                      headerCategory: 'wavelength',
-                      headerValue: 'value',
```

Comment:
How is the csv file created currently? Is this export portion of the code responsible for that? Just trying to understand how this was being done previously.

Reply:
I looked into it more and it makes sense now. This implementation looks good. It is just annoying that the Svelte csv download feature can not be customized.